src/content/docs/installation/boot_managers.md

```diff
@@ -18,6 +18,25 @@ handles the task of loading an OS kernel into memory, often along with support f
 For a more detailed explanation and the reason both terms are often used interchangeably,
 please see "[Managing EFI Boot Loaders for Linux: Basic Principles](https://www.rodsbooks.com/efi-bootloaders/principles.html)" by Rod Smith.
 
+Here is the feature comparison table for the boot managers, accompanied by additional details and recommendations:
+
+### **Feature Comparison Table**
+
+| **Feature**                | **systemd-boot**                     | **rEFInd**                            | **GRUB**                           |
+|----------------------------|---------------------------------------|---------------------------------------|------------------------------------|
+| **UEFI Support**            | ✅ Yes                               | ✅ Yes                               | ✅ Yes                            |
+| **BIOS Support**            | ❌ No                                | ❌ No                                | ✅ Yes                            |
+| **Autodetection**           | ❌ Limited (manual configuration)    | ✅ Full autodetection                | ✅ Autodetection (regenerates config) |
+| **Theming/Customization**   | ❌ Minimal                           | ✅ Extensive                          | ✅ Extensive                      |
+| **Configuration Complexity**| Simple                              | Moderate                             | Complex                          |
```

Comment:
I don't really agree with this either, but I would need to fact check on rEFind's configuration

Reply:
I can look into this, but since I don’t personally use rEFInd, it may take some time.